tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -27,10 +27,26 @@ ResourceInfo:
 
 Evolution:
   InitialTime: 0.0
-  InitialTimeStep: 0.01
+  InitialTimeStep: 0.0002
+  # This is the smallest interval we'd need to observe time step/constraints. If
+  # you need it smaller you can edit it, but make sure to change the slab
+  # intervals in the EventsAndTriggers
+  InitialSlabSize: 0.001
```

Comment:
You shouldn't restrict the slab size to line up with observations, particularly not to something this small.  If you need more frequent output use dense output instead of forcing extra RHS evaluations.

Reply:
This was sort of arbitrarily chosen to make the test not take forever. This wouldn't be very realistic for an actual evolution. Maybe I should change the comment to make it clear that you should change the slab size if you're using this yaml or use dense output if you want more frequent output.

Reply:
Yeah, having the value small for the test is fine, but the comment shouldn't recommend doing observations that way.